src/olympia/amo/utils.py

```diff
@@ -666,12 +666,14 @@ def pngcrush_image(src, **kw):
     return False
 
 
-def resize_image(source, destination, size=None, format='png'):
+def resize_image(source, destination, size=None, *, format='png', quality=80):
```

Comment:
Sorry if i am wrong but why you setting it to 80? is it going to be ignored for jpg (quality=80)?

Reply:
80 is the default value of `quality` if the kwarg isn't provided.  As to "why 80?" - 80 is generally the best compromise between filesize and not losing any noticeable quality. Mainly though, it's been the default we've used for years for jpegs so I didn't want to regress the use of `resize_image` elsewhere.

Reply:
thank you so much @eviljeff got it, sorry for noise